From: Sophie Brun <sophie@offensive-security.com>
Date: Tue, 13 Apr 2021 11:23:44 +0200
Subject: change output dir location

output dir is by default in the application directory. It's not
compatible with users rights. We change the output directory to
~/.commix/output
Forwarded: not-needed
Last-Update: 2021-09-14
---
 src/core/main.py      | 2 +-
 src/utils/settings.py | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/core/main.py b/src/core/main.py
index 2322969..59d70f0 100644
--- a/src/core/main.py
+++ b/src/core/main.py
@@ -345,7 +345,7 @@ def logs_filename_creation():
     os.stat(output_dir)
   except:
     try:
-      os.mkdir(output_dir)   
+      os.makedirs(output_dir)   
     except OSError as err_msg:
       try:
         error_msg = str(err_msg).split("] ")[1] + "."
diff --git a/src/utils/settings.py b/src/utils/settings.py
index 8d591d6..c1ee38d 100755
--- a/src/utils/settings.py
+++ b/src/utils/settings.py
@@ -327,7 +327,7 @@ GIT_URL = "https://github.com/commixproject/" + APPLICATION + ".git"
 ISSUES_PAGE = "https://github.com/commixproject/" + APPLICATION + "/issues/new"
 
 # Output Directory
-OUTPUT_DIR = ".output/"  
+OUTPUT_DIR = os.path.expanduser("~/.commix/output/")
 
 # Output file name
 OUTPUT_FILE_NAME = "logs"
@@ -1027,4 +1027,4 @@ PAGE_COMPRESSION = None
 # Force usage of given HTTP method (e.g. PUT).
 HTTP_METHOD = ""
 
-# eof
\ No newline at end of file
+# eof
